docs/hardware.rst

```diff
@@ -382,6 +383,36 @@ NUC before it can be used. We recommend:
 -  1x memory kit of compatible 2x8GB sticks
    -  You can put one 8GB memory stick in each of the servers.
 
+.. _nuc10_recommendation:
+
+Intel 10th-gen NUC
+~~~~~~~~~~~~~~~~~~
+We have tested and can recommend the `Intel NUC10i5FNH <https://ark.intel.com/content/www/us/en/ark/products/189239/intel-nuc-10-performance-kit-nuc10i5fnh.html>`__.
+It provides two storage options: M.2 SSD storage and a 2.5" secondary storage
+option (SSD or HDD).
+
+The NUC10's AX201 wireless hardware is not removable. Before installation of the
+RAM and storage, we recommend that you disconnect the wireless antennae leads
+from the AX201 component. They're the black and gray wires highlighted in
+the red box in the picture. Cover the free ends with electrical tape
+after disconnecting them.
+
+|NUC10 leads|
+
+Before the initial OS installation, boot into the BIOS by pressing **F2** at
+startup and adjust the system configuration:
```

Comment:
Note that while it's branded to look very similar, the Aptio BIOS uses a slightly different menu structure than the Visual BIOS for other NUC models.